sssd_test_framework/utils/sssd.py

```diff
@@ -135,6 +135,10 @@ def async_start(
         if apply_config:
             self.config_apply(check_config=check_config, debug_level=debug_level)
 
+        # Also stop kcm
+        if service == "sssd":
+            self.svc.async_stop("sssd-kcm.service")
```

Comment:
> What test are failing?

sudo tests (tests/test_sudo.py).

> 
> KCM is only needed for KCM tests, otherwise it should not be needed to restart it. Wouldn't it be better to call `start("sssd-kcm")` in these tests instead of just `start()`?

Several tests restart SSSD and they end up restarting also KCM because this mechanism already exists.